### DISCUSSION TITLE
Deployments via Python SDK to support prefect.yaml features

### DISCUSSION THREAD

**Discussion by marcm-ml** [PrefectHQ/prefect#10505] [Q&A]
Are there plans to unify the Python Deployment SDK and the CLI? Because I can see huge discrepancies in these two ways of creating a deployment?

**Comment by discdiver** [comment]
We recently introduced flow.serve, which allows you to create a deployment and server quickly in Python. See the [updated tutorial](https://docs.prefect.io/latest/tutorial/deployments/#create-a-deployment) to get started with it. There is more work being doing in that area. 🙂
